### PR TITLE
DISTPG-386 Release notes 13.6

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   .github/workflows/main.yml
	new file:   docs/release-notes-v13.6.md
	modified:   docs/release-notes.md
	modified:   mkdocs-base.ym

### CODE REVIEW

File: docs/release-notes-v13.6.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+| [pgBackRest](https://pgbackrest.org/)           | 2.37    | a backup and restore solution for PostgreSQL       |
+|[`pgBadger`](https://github.com/darold/pgbadger) | 11.7       | a fast PostgreSQL Log Analyzer.|
+|[`pgBouncer`](https://www.pgbouncer.org/) | 1.16.1 | lightweight connection pooler for PostgreSQL|
+| [pg_repack](https://github.com/reorg/pg_repack) | 1.4.7   | rebuilds PostgreSQL database objects           |
+| [pg_stat_monitor](https://github.com/percona/pg_stat_monitor)| 1.0.0 - rc.1 | collects and aggregates statistics for PostgreSQL and provides histogram information.       |
+| [PostgreSQL Common](https://packages.debian.org/sid/percona-postgresql-common)| 237 | PostgreSQL database-cluster manager. It provides a structure under which multiple versions of PostgreSQL may be installed and/or multiple clusters maintained at one time.|
+|[`wal2json`](https://github.com/eulerto/wal2json) |2.4        | a PostgreSQL logical decoding JSON output plugin.|
+
+Percona Distribution for PostgreSQL also includes the following packages:
+- `llvm` 12.0.1 packages for Red Hat Enterprise Linux 8 / CentOS 8. These fix compatibility issues with LLVM from upstream.
```
Comment (LenzGr) [inline docs/release-notes-v13.6.md]:
```suggestion
- `llvm` 12.0.1 packages for Red Hat Enterprise Linux 8 (and compatible derivatives). These fix compatibility issues with LLVM from upstream. 
```

File: docs/release-notes-v13.6.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+|[`wal2json`](https://github.com/eulerto/wal2json) |2.4        | a PostgreSQL logical decoding JSON output plugin.|
+
+Percona Distribution for PostgreSQL also includes the following packages:
+- `llvm` 12.0.1 packages for Red Hat Enterprise Linux 8 / CentOS 8. These fix compatibility issues with LLVM from upstream. 
+- supplemental ETCD packages which can be used for setting up Patroni clusters. These packages are available for the following operating systems:
+
+|  Operating System   |Package               | Version | Description        |
+| ------------------- | ---------------------| --------| -------------------|
+| CentOS 7            |`python3-python-etcd` | 0.4.3   | A Python client for ETCD     |
+| CentOS 8            | `etcd`               | 3.3.11  | A consistent, distributed key-value store|
```
Comment (LenzGr) [inline docs/release-notes-v13.6.md]:
```suggestion
| RHEL 7            |`python3-python-etcd` | 0.4.3   | A Python client for ETCD     |
| RHEL 8            | `etcd`               | 3.3.11  | A consistent, distributed key-value store|
```

File: docs/release-notes-v13.6.md (hunk 1 above)
Comment (nastena1606) [reply docs/release-notes-v13.6.md]: Fixed

File: docs/release-notes-v13.6.md (hunk 2 above)
Comment (nastena1606) [reply docs/release-notes-v13.6.md]: Fixed

File: mkdocs-base.yml [hunk 3]
```diff
@@ -83,7 +83,7 @@ plugins:
   with-pdf: # https://github.com/orzih/mkdocs-with-pdf
           output_path: '_pdf/PerconaDistributionPostgreSQL-13.pdf'
           cover_title: 'Percona Distribution for PostgreSQL Documentation'
-          cover_subtitle: 13.5 (December 7, 2021)
+          cover_subtitle: 13.6 (March 21, 2022)
```
Comment (EvgeniyPatlan) [inline mkdocs-base.yml]: date is in the past

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

**Pull request merged**